motionplan/tpspace/ptgSideS.go

```diff
@@ -0,0 +1,106 @@
+package tpspace
+
+import (
+	"math"
+
+	"go.viam.com/rdk/referenceframe"
+	"go.viam.com/rdk/spatialmath"
+)
+
+const defaultCountersteer = 1.5
+
+// ptgDiffDriveSideS defines a PTG family which makes a forwards turn, then a counter turn the other direction, and goes straight.
+// This has the effect of translating to one side or the other without orientation change.
+type ptgDiffDriveSideS struct {
+	maxMMPS      float64 // millimeters per second velocity to target
+	maxRPS       float64 // radians per second of rotation when driving at maxMMPS and turning at max turning radius
+	r            float64 // turning radius
+	countersteer float64 // scale the length of the second arc by this much
```

Comment:
Could make this another circle ptg

Reply:
I thought that since this was an S shape we could compose it from two circle ptgs instead of the one we have but maybe thats not possible

Reply:
That's pretty much what's happening here, we just use the same circle PTG for both arcs.

Reply:
Ah ok I think I was missing that part